hugr-core/src/extension/op_def.rs

```diff
@@ -217,11 +268,12 @@ impl From<CustomValidator> for SignatureFunc {
 }
 
 impl SignatureFunc {
-    fn static_params(&self) -> &[TypeParam] {
-        match self {
+    fn static_params(&self) -> Result<&[TypeParam], SignatureError> {
+        Ok(match self {
             SignatureFunc::PolyFuncType(ts) => ts.poly_func.params(),
```

Comment:
aaiieeee is this one even correct? "static_params" are the ones to pass to binary compute_signature, so if there isn't such, I think this should be the empty list

Reply:
yeah I found these functions aren't actually used (therefore aren't tested...) will add an issue.